File: 151004/Stahnov/rest/src/main/java/com/distributed_computing/rest/bean/Tweet.java
package com.distributed_computing.rest.bean;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
public class Tweet {
    int id;
    int creatorId;
    String title;
    String content;
    LocalDateTime created;
    LocalDateTime modified;
}

File: 151003/Alekseeva/TaskKafka/Module.Publisher/Models/Tweet.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;
namespace Publisher.Models;

[Table("tbl_tweets")]
[Index(nameof(Title), IsUnique = true)]
public class Tweet
{
    public long Id { get; set; }

    [ForeignKey("Creator")] public long? CreatorId { get; set; }

    [MaxLength(32)] public string Title { get; set; } = string.Empty;
    [MaxLength(2048)] public string Content { get; set; } = string.Empty;

    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public DateTime? Created { get; init; }

    [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
    public DateTime? Modified { get; set; }

    public Creator? Creator { get; set; }
    public List<Tag> Tags { get; } = [];
}